### PR TITLE
Feature/monte carlo

### PR DESCRIPTION
**Pull request by EvanY14**

### CODE REVIEW

File: Simulation/6DOF_RK4/plotter/analysis.py
Comment (parthshrotri) [inline Simulation/6DOF_RK4/plotter/analysis.py:1]: This merge shouldn't delete the previous code, I was thinking Montecarlo would be a separate analysis file, why's this being deleted?

**Review by parthshrotri — CHANGES_REQUESTED** [review]
Analysis.py has a lot of deleted code that seems like it should be kept, maybe this was deleted when you created the trajectory graph?

File: Simulation/6DOF_RK4/plotter/analysis.py
Comment (EvanY14) [reply Simulation/6DOF_RK4/plotter/analysis.py:1]: Yeah I think I accidentally got rid of it while I was resolving the merge conflicts

**Review by parthshrotri — APPROVED** [review]
I approve this, the first PR submitted by Evan Yu as glorious leader

**Pull request merged**